### ISSUE TITLE
snpeff build stackoverflow error

### ISSUE THREAD

**Issue by afkoeppel** [tseemann/snippy#72]
I'm getting a stackoverflow error at the snpeff build step.  
My reference file is the Mycobacterium tuberculosis complete genome in genbank (full) format (downloaded from http://www.ncbi.nlm.nih.gov/nuccore/448814763?report=genbank).  

Works fine when I use the fasta as a reference, but  I'd like to get the annotation information from using the genbank reference.

My original command was:
snippy --cpus 16 --outdir testgb_snps --ref Myc_tub_H37Rv_NC_000962.gb --pe1 136MDR_S6_R1.fq.gz --pe2 136MDR_S6_R2.fq.gz

The command that threw the error was:
snpEff build -c reference/snpeff.config -dataDir . -gff3 ref
java.lang.StackOverflowError
        at java.lang.Character.toLowerCase(Character.java:6338)
        at java.lang.Character.toLowerCase(Character.java:6309)
        at java.lang.String.toLowerCase(String.java:2581)
        at java.lang.String.toLowerCase(String.java:2670)
        at ca.mcgill.mcb.pcingola.interval.GffMarker.hasAttr(GffMarker.java:207)
etc.

Software versions:
snippy 3.1
snpeff 4.2
java 1.8.0_91
bwa 0.7.12
samtools 1.3

Any advice would be appreciated.

**Comment by afkoeppel** [comment]
Update:  The snpEff build command seems to work just fine on the test data in the snippy directory (ran test3 with no issues).  I assumed therefore that it was something off with my genbank file.  I tried re-downloading the genome from another location but the file was the same so that didn't make a difference.
I also tried downloading the same genome in EMBL format from EMBL, but I got the same Stackoverflow error.
Attaching my genbank and embl files here in case that helps.

Genbank
[Myc_tub_H37Rv_NC_000962.gb.gz](https://github.com/tseemann/snippy/files/318702/Myc_tub_H37Rv_NC_000962.gb.gz)

EMBL
[GCA_000195955-2.AL123456.txt.gz](https://github.com/tseemann/snippy/files/318704/GCA_000195955-2.AL123456.txt.gz)
